.github/workflows/docs/conf.py

```diff
@@ -10,9 +10,24 @@
     "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.mathjax",
+    "sphinx_copybutton",
 ]
 
-html_theme = "sphinx_rtd_theme"
+pygments_style = "borland"
+
+html_theme = "sphinx_book_theme"
+
+html_theme_options = {
+    "repository_url": "https://github.com/CQCL/pytket-qiskit",
```

Comment:
Ah the repository URL needs to be changed here.

Reply:
done

Reply:
Thank you for spotting!